src/App.tsx

```diff
@@ -1,10 +1,53 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Filter } from './components/Filter';
+import { Notifications } from './components/Notifications';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { ErrorText } from './types/ErrorText';
+import { Filters } from './types/Filters';
+
+const USER_ID = 11213;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loadedTodo, setLoadingTodo] = useState<number | null>(null);
+  const [fieldOperation, setFieldOperation] = useState(ErrorText.empty);
+  const [selectedFilter, setSelectedFilter] = useState<Filters>(Filters.all);
+
+  const activeTodos = todos.filter(t => t.completed === false);
+  const completedTodos = todos.filter(t => t.completed === true);
+
+  const filteredTodos = (filterChar: Filters) => {
+    return todos.filter(todo => {
+      switch (filterChar) {
+        case 'all':
+          return true;
+
+        case 'active':
```

Comment:
Use a enum for these magic words
<img width="389" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/64750958/ce254b16-a730-40a0-8090-c96006884a3a">